app/views/error-404.nunjucks

```diff
@@ -1,13 +1,16 @@
 {% extends 'layout.nunjucks' %}
 
+{% set message = 'Page not found' %}
+
+{% block pageTitle %} {{ message }} {% endblock %}
+
 {% block header %}
-Page not found  
+  {{ message }}
 {% endblock %}
 
 {% block content %}
   <div class="reading-width">
-		<p>If you have entered a web address check it was correct. You can <a href="//nhs.uk/">browse from the NHS.UK home page</a>.</p>
-		<p><a href="javascript:history.go(-1)" class="link-back">Back</a></p>
+    <p>If you have entered a web address check it was correct. You can <a href="//nhs.uk/">browse from the NHS.UK home page</a>.</p>
```

Comment:
The href here needs to be `http://www.nhs.uk` given the current site doesn't work over `https`